.github/workflows/test.yml

```diff
@@ -105,9 +105,12 @@ jobs:
       - name: Test
         run: |
           GO_TAGS="stablediffusion tts" make test
+      - name: Setup tmate session if tests fail
```

Comment:
ouch, better to not commit this - if something fails then tmate will be hanging in there waiting for someone to connect and will exhaust all the CIs..

Reply:
@mudler What do you think about using
`timeout-minutes: 5` instead?
Apparently we can tune how long it sits around waiting - my thought locking these in place on every failure was that's easier than the alternative the docs for the action recommend, which is to use manual dispatches with an event like `if: ${{ github.event_name == 'workflow_dispatch' && inputs.debug_enabled }}`

I'm not wed to any particular wait duration... I'd just like it to be longer than 1 minute so I have a chance to notice and connect :D

Reply:
I also only put it on these two test runners rather than any of the builders to try and keep the wait pressure down FYI